tests/test_frame.py

```diff
@@ -81,9 +82,44 @@ def test_FrameConvert(self):
         nframe = astshim.Frame(2)
         fset = frame.convert(nframe)
         self.assertEqual(fset.getClass(), "FrameSet")
+
+        # the conversion FrameSet should contain two frames
+        # connected by a unit mapping with 2 axes
+        self.assertEqual(fset.getNframe(), 2)
+        self.assertEqual(fset.getNin(), 2)
+        self.assertEqual(fset.getNout(), 2)
+        indata = np.array([
+            [1.1, 2.2],
+            [-43.5, 1309.31],
+        ])
+        outdata = fset.tranForward(indata)
+        assert_allclose(outdata, indata)
```

Comment:
The test for `Frame.convert` is quite weak, but I suggest another ticket for that. In general tests of the more complicated methods and approximation methods tend to be trivial, intended more as a sanity check that the wrapping was done correctly than as a torture test of the algorithm. It would be good to beef up the tests.